gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -152,7 +152,7 @@ public Void call() throws JobException {
         GobblinClusterConfigurationKeys.JOB_ALWAYS_DELETE,
         "false");
 
-    try {
+    try { //TODO: what is really the difference ?
```

Comment:
Why is this relevant to this PR?

Reply:
oh this got into this by mistake, it was my own code comment. will remove it.